pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -575,6 +582,30 @@ protected void startStatsUpdater(int statsUpdateInitialDelayInSecs, int statsUpd
         updateRates();
     }
 
+    protected void initializeHealthChecker() {
+        ServiceConfiguration config = pulsar().getConfiguration();
+        if (config.getHealthCheckMetricsUpdateTimeInSeconds() > 0) {
+            this.healthChecker = OrderedScheduler.newSchedulerBuilder()
+                    .name("pulsar-health-checker")
+                    .numThreads(1)
+                    .build();
+            int healthCheckFreqInSecs = config.getHealthCheckMetricsUpdateTimeInSeconds();
+            int healthCheckInitialDelayInSecs = config.getHealthCheckMetricsUpdateTimeInSeconds();
```

Comment:
`healthCheckFreqInSecs` and `healthCheckInitialDelayInSecs` are same variable. we should use only 1 `healthCheckFreqInSecs`. please remove duplicate.